src/map.rs

```diff
@@ -255,9 +255,10 @@ impl Map {
                 0, 0, screen_w, screen_h,
             );
 
+            let opacity = (hit_effect_timer/constants::HIT_SEQUENCE_AMOUNT
+                    * constants::MAX_RED_ALPHA * 255.) as u8;
             canvas.set_draw_color((
-                200, 0, 0, (hit_effect_timer/constants::HIT_SEQUENCE_AMOUNT
-                    * constants::MAX_RED_ALPHA * 255.) as u8
+                200, 0, 0, opacity
```

Comment:
This could be a single line but not super important

Reply:
Fixed